projects/demo/server.ts

```diff
@@ -1,5 +1,5 @@
 import { APP_BASE_HREF } from '@angular/common';
-import { CommonEngine } from '@angular/ssr';
```

Comment:
Shouldn't we regenerate this file to use the new app engine?
Not a big deal since we don't use it anyway, but still

Reply:
Yeah I ignored for now as it isn't used...